src/pages/components/DisplayGapsPercentage.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { TEXTS } from '../../resources/texts'
+import './DisplayGapsPercentage.scss'
+
+function DisplayGapsPercentage(gapsPrecentage: number | undefined) {
```

Comment:
let's make it a Component
```suggestion
function DisplayGapsPercentage({gapsPrecentage}: {gapsPrecentage: number | undefined}) {
```

Reply:
I agree, my mistake. I will change it to be a component as you told me before as well.